migrations/utils/init_settings/initNodeComponentsSettings.js

```diff
@@ -0,0 +1,23 @@
+const {
```

Comment:
I would change the file name to initComponentVersions.js or similar to be consistent with the contract functions.